### DISCUSSION TITLE
Support elixir's OTP versioning scheme

### DISCUSSION THREAD

**Discussion by jprinaldi** [renovatebot/renovate#28740] [Suggest an Idea]
### Tell us more.

I install `elixir` versions using `asdf`. Currently, the latest `elixir` version is `1.16.2`. However, that version is built against OTP 24, even though there are versions that also support OTP 25 and OTP 26.

Because of this, I installed `1.16.2-otp-26`, to make sure the latest OTP features are supported. However, after doing that, Renovate opened a PR suggesting "updating" that version back to `1.16.2`, which is a definite downgrade.

It would be very useful if Renovate supported this versioning scheme.

References:
https://github.com/asdf-vm/asdf-elixir?tab=readme-ov-file#elixir-precompiled-versions
https://github.com/hexpm/bob/blob/main/README.md#elixir-builds

**Comment by rarkins** [comment]
Set versioning=docker for this package

**Reply by jprinaldi** [reply]
Thanks. It seems to me like this would be the behavior most people would expect in this situation. What's the reason behind Renovate not behaving like this by default?

**Reply by rarkins** [reply]
Asdf rules are mostly community contributed, you're welcome to submit a change
